src/Controllers/OAuthController.php

```diff
@@ -130,14 +136,22 @@ private function checkExistenceUser(array $find = []): bool
         return $findUser !== null;
     }
 
-    private function syncingUserInfo(array $find = [], array $updateFildes = []): int
+    /**
+     * Syncs user information based on provided fields.
+     *
+     * @param array<string, mixed> $find         Array containing criteria to find the user
+     * @param array<string, mixed> $updateFields Fields to update for the user
```

Comment:
```suggestion
     * @param array<string, string> $find         Array containing criteria to find the user
     * @param array<string, string|null> $updateFields Fields to update for the user
```
Using `mixed ` doesn't explain the code well enough. How are you with the change below?